### ISSUE TITLE
Aim only for golden collection

### ISSUE THREAD

**Issue by ericBG** [ko-vasilev/Hearthstone-Collection-Tracker#31]
Not sure how to write it in the title, but basically what I do is I keep the golden version of everything if I get it, and dust the unnecessary normals. So say I have 1 golden Power of the Wild and 2 normal ones, I would dust a normal one. Is there any way to incorporate this into the application?

**Comment by ko-vasilev** [comment]
Hey, yes it is doable, I'll consider it in the future.

**Comment by Zupami** [comment]
Yeah this is exactly the feature that I'm looking for as well, would be really cool to see!
When a new update comes out, will we be notified through HDT? Or do we just have to check back here? Or maybe you got some other way keeping us up to date?

**Comment by ko-vasilev** [comment]
When new update is available you will be notified through Collection Tracker main window.
